packages/lesswrong/components/tagging/subforums/TagSubforumPage2.tsx

```diff
@@ -369,19 +341,14 @@ const TagSubforumPage2 = ({classes}: {
     >
       <HeadTags description={headTagDescription} />
       {hoveredContributorId && <style>{`.by_${hoveredContributorId} {background: rgba(95, 155, 101, 0.35);}`}</style>}
-      {tag.bannerImageId && (
-        <div className={classes.imageContainer}>
-          <CloudinaryImage2 publicId={tag.bannerImageId} fullWidthHeader />
-        </div>
-      )}
-      <div className={tag.bannerImageId ? classes.contentGivenImage : ""}>
-        <RightSidebarColumn
-          sidebarComponents={rightSidebarComponents[tab]}
-          header={headerComponent}
-        >
-          {tabComponents[tab]}
-        </RightSidebarColumn>
-      </div>
+      <RightSidebarColumn
+        titleComponent={titleComponent}
+        bannerImageId={tag.bannerImageId}
+        headerComponent={headerComponent}
+        sidebarComponents={rightSidebarComponents[tab]}
+      >
+        {tabComponents[tab]}
+      </RightSidebarColumn>
```

Comment:
Did this component used to be just the right sidebar, and now contains basically all the main page contents? If so, would be nice to rename it

Reply:
I've renamed it, perhaps controversially, to `SubforumLayout`. This is just for consistency with the rest of the code where `isSubforum` is the flag that decides whether this page is shown